README.md

```diff
@@ -12,55 +12,25 @@ Contents:
 - [Hevm](./src/hevm) - Ethereum evaluator. Fully complient Haskell EVM implementation.
 - [Ethsign](./src/ethsign) - sign Ethereum transactions from a local keystore.
 
-## Easy install
-All tools and dependencies can be automaticaly installed by running:
-```sh
-curl https://dapp.tools/install | sh
-```
+## Instalation
 
-You _may_ also need to add the nix binaries to your path:
+Install `nix` if you haven't already:
 
 ```sh
-export PATH="$PATH:$HOME/.nix-profile/bin"
-```
-
-## Manual install
-For a manual install, please follow the instructions below.
-
-### Install Nix
-
-Make sure you are using Nix v2
-
-    curl https://nixos.org/nix/install | sh
-
-    # Run this or login again to use Nix
-    . "$HOME/.nix-profile/etc/profile.d/nix.sh"
-
-### Setup Cachix (optional)
-
-If you want to cache the binary builds across several computers, Nix
-provides several ways of doing this, the most convenient being
-the [Cachix](https://cachix.org) service.
+curl https://nixos.org/nix/install | sh
 
-We maintain a public cache which you can use by installing Cachix
-
-    nix-env -if https://github.com/cachix/cachix/tarball/master --substituters https://cachix.cachix.org --trusted-public-keys cachix.cachix.org-1:eWNHQldwUO7G2VkjpnjDbWwy4KQ/HNxht7H4SSoMckM=
-
-then running
-
-    cachix use dapp
-
-followed by restarting your Nix daemon (if on a multi-user Nix installation)
-
-### Install tools
-
-To install tools from this repository, you need to run
+# Run this or login again to use Nix
+. "$HOME/.nix-profile/etc/profile.d/nix.sh"
+```
 
-    git clone --recursive https://github.com/dapphub/dapptools $HOME/.dapp/dapptools
-    nix-env -f $HOME/.dapp/dapptools -iA dapp seth solc hevm ethsign
+Then install dapptools:
 
-which will build and install these tools into your Nix profile, optionally
-fetching the binaries from the cache if you have configured it.
+```
+nix-env -A seth -A dapp -A hevm \
```

Comment:
```suggestion
nix-env -iA seth dapp hevm \
```

Reply:
Traditionally we also include solc, but leaving it out seems okay too

Reply:
If we do install it, we should at least check if the user already has it to make sure we are not silently overwriting previous versions

Reply:
> Traditionally we also include solc, but leaving it out seems okay too

I think that I would prefer to leave it out, exactly to avoid having to check to see if it is installed already.

Reply:
Usually, overlays choose different attribute names (e.g. `emacsUnstable` or `firefox-nightly`), we might want to do the same.

The binary would still be available as `solc`, this only affects how you install it.

Reply:
But I guess that's a [separate issue](#380).